templates/taxi/car_detail.html

```diff
@@ -16,7 +16,23 @@ <h1>Drivers</h1>
   <hr>
   <ul>
     {% for driver in car.drivers.all %}
-      <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
+      <li>
+        <p>Username: {{ driver.username }}</p>
+        <p>Full name: {{ driver.first_name }} {{ driver.last_name }}</p>
+        <p>License number: {{ driver.license_number }}</p>
+      </li>
     {% endfor %}
   </ul>
+  
+  {% if user in car.drivers.all %}
+    <form method="post" action="{% url 'taxi:car-remove-driver' car.pk %}">
+      {% csrf_token %}
+      <button type="submit" class="btn btn-danger">Delete me from this car</button>
+    </form>
```

Comment:
you use here backward relationship but did you fetched it from db? because otherwise here will be N+1 issue